thirdparty/build-thirdparty.sh

```diff
@@ -371,19 +367,15 @@ build_gperftools() {
     check_if_source_exist $GPERFTOOLS_SOURCE
     cd $TP_SOURCE_DIR/$GPERFTOOLS_SOURCE
 
-    OLD_FLAGS=$CFLAGS
-    CFLAGS="-O3 -fno-omit-frame-pointer -fPIC -g"
-
     if [ ! -f configure ]; then
         ./autogen.sh
     fi
 
     LDFLAGS="-L${TP_LIB_DIR}" \
+    CFLAGS="-O3 -fno-omit-frame-pointer -fPIC -g" \
     ./configure --prefix=$TP_INSTALL_DIR/gperftools --disable-shared --enable-static --disable-libunwind --with-pic --enable-frame-pointers
     make -j$PARALLEL
     make install
```

Comment:
Why not need to reset `CFLAGS` here?

Reply:
can either unset and export a new flags or use one line format to temporarily change the CFLAGS vaule.